packages/devtools_extensions/README.md

```diff
@@ -154,15 +154,23 @@ To use a real DevTools environment, you will need to perform a series of setup s
 real DevTools environment. Build your flutter web app and copy the built assets from
 `your_extension_web_app/build` to your pub package's `extension/devtools/build` directory.
 
-> Use the `build_helper.sh` script to help with this step. (TODO(kenz): create and add link to helper script).
+Use the `build_extension` command from `package:devtools_extensions` to help with this step.
+```
+dart pub global activate devtools_extensions;
```

Comment:
This doesn't work:

```
$ dart pub global activate devtools_extensions
Because pub global activate depends on devtools_extensions any which requires the Flutter SDK, version solving failed.

Flutter users should run `flutter pub get` instead of `dart pub get`.
```